includes/general.php

```diff
@@ -1052,7 +1052,13 @@ function pods_shortcode_run( $tags, $content = null ) {
 
 		return $pod->form( $tags['fields'], $tags['label'], $tags['thank_you'] );
 	} elseif ( ! empty( $tags['field'] ) ) {
-		if ( empty( $tags['helper'] ) ) {
+		if ( ! empty( $tags['template'] ) ) {
+			$return  = '';
+			$related = $pod->field( $tags['field'], array( 'output' => 'find' ) );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 1d0149ad9aad33436ad8ca94d338441b</sub>